components/ai_chat/core/browser/engine/oai_api_client.cc

```diff
@@ -122,6 +125,22 @@ void OAIAPIClient::PerformRequest(
   }
 }
 
+std::optional<std::string> OAIAPIClient::ExtractErrorMessage(
```

Comment:
This could be moved to an anonymous namespace at the top of the file

Reply:
moved